portal/templates/initial_queries_macros.html

```diff
@@ -8,31 +8,99 @@ <h4 class="tnth-headline">{{ _("Terms of Use") }}</h4>
-                </label>
-                <br/>
-                <label class="general-tou" data-agree="false" data-type="terms" data-tou-type="website terms of use" data-core-data-type="website_terms_of_use" data-url="{{terms['url'] if terms}}">
-                    <i class="fa fa-square-o fa-lg terms-tick-box"></i><span class="default-tou-text terms-tick-box-text">{{_("I have read the information notice above and consent and agree to the processing of my personal information (including my health information) on the terms described in this Consent and Terms and Conditions.")}}</span>
-                    &nbsp;&nbsp;
-                    {% trans privacy_url=url_for('portal.privacy', disableLinks="true"), terms_url=url_for('portal.terms_and_conditions', disableLinks="true") %}
-                    <span class="custom-tou-text">I have read the website <a href="{{privacy_url}}" type="terms" class="required-link" data-core-data-subtype="privacy_policy" data-tou-type="privacy policy">privacy policy</a> and <a href="{{terms_url}}"  class="required-link">terms</a></span>
-                    {% endtrans %}
-                </label>
-                <br/>
-                <label class="custom-tou tnth-hide" data-agree="false" data-type="terms" data-tou-type="stored website consent form" data-core-data-type="stored_website_consent_form" data-url="{{terms['url'] if terms}}">
-                  <i class="fa fa-square-o fa-lg terms-tick-box"></i> <span class="terms-tick-box-text">{{ _("I have read and/or gone through required information to the subject and have completed the required consent to the use of the TrueNTH website tool and have created an electronic copy of this declaration and have stored said copy.") }}</span>
-                </label>
+               {% set tou_url = terms['url'] if terms else '' %}
+               <!-- note these checkboxes are selectively hidden based on user role-->
+               {% if user.has_role(ROLE.STAFF) or user.has_role(ROLE.STAFF_ADMIN) %}
+                    <label class="terms-label tnth-hide">
+                        <i class="fa fa-square-o fa-lg terms-tick-box"></i>
+                        {% trans privacy_url=url_for('portal.privacy', disableLinks="true"), terms_url=url_for('portal.terms_and_conditions', disableLinks="true") %}
+                        <span class="terms-tick-box-text">&nbsp;By checking this box, I confirm that I have read and accept the website <a
+                          href="{{privacy_url}}" class="required-link">privacy policy</a> and <a
+                          href="{{terms_url}}" class="required-link">terms</a></span>
+                        {% endtrans %}
+                        <input type="hidden"
+                          data-type="terms"
+                          data-agree="false"
+                          data-core-data-type="privacy_policy"
+                          data-tou-type="privacy policy"
+                          data-url="{{privacy_url}}"/>
```

Comment:
Do these template variables (`privacy_url`, `tou_url`) really work outside the `{% endtrans %}` block?

Reply:
thank you for catching this Ivan!  I think fixed another place where it has the same issue. let me know if you find issue with my fix though.